SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs

```diff
@@ -356,16 +356,13 @@ private Task SendErrorAttachmentsAsync(Guid errorId, IEnumerable<ErrorAttachment
                 {
                     attachment.Id = Guid.NewGuid();
                     attachment.ErrorId = errorId;
-                    try
+                    if (!attachment.ValidatePropertiesForAttachment())
```

Comment:
Why not use the Validate method (and just modify it?) Is it because it is missing something like device at this point? If that's the reason, can you just add a comment explaining?

Reply:
Also, maybe invert the `if`

Reply:
Yes, the device will be initialized later by channel.
We have similar way in iOS (separate validate function).

Let's keep validate **base** log class as is and just add a comment here

Reply:
Also the code is generated and swagger spec is not good enough for that for some models.

Reply:
@AnastasiaKubova can you address https://github.com/microsoft/appcenter-sdk-dotnet/pull/1021/files#r301215312 in another PR? I unblocked this one as good enough to unblock test app review.

Reply:
Ok